lib/adBlockRustUtils.js

```diff
@@ -2,25 +2,38 @@ const { uBlockResources } = require('adblock-rs')
 
 const path = require('path')
 const fs = require('fs')
+const request = require('request')
 
 const uBlockLocalRoot = 'submodules/uBlock'
 const uBlockWebAccessibleResources = path.join(uBlockLocalRoot, 'src/web_accessible_resources')
 const uBlockRedirectEngine = path.join(uBlockLocalRoot, 'src/js/redirect-engine.js')
 const uBlockScriptlets = path.join(uBlockLocalRoot, 'assets/resources/scriptlets.js')
 
+const braveResourcesUrl = 'https://raw.githubusercontent.com/brave/adblock-resources/master/dist/resources.json'
```

Comment:
Sounds good that this is master, but I think security team will want to review the things in `adblock-resources` if we keep this at master and not a specific changeset. cc @diracdeltas

Reply:
@diracdeltas if you'd like, i'd be happy to do sec review of these resources too (at least for the ones I dont write), in the same way I'm doing for upstream uBO resources